src/Context/Cart/Cart-Context.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { createContext, useContext, useState } from "react";
+import { useNavigate } from "react-router-dom";
+const CartContext = createContext();
+
+const CartProvider = ({ children }) => {
+  const [cart, setCart] = useState([]);
+  const [increaseQuantity, setIncreasedQuantity] = useState(1);
+  const [decreaseQuantity, setDecreasedQuantity] = useState(1);
+
+  const cartToggleHandler = (product) => {
```

Comment:
Suggestion: Changes this is to "add to cart "or something because this is toggling the item in the cart.